packages/maskbook/src/plugins/Polls/UI/Polls.tsx

```diff
@@ -39,15 +39,15 @@ const useStyles = makeStyles((theme) =>
             position: 'absolute',
             top: '0',
             left: '0',
-            zIndex: 100,
```

Comment:
I'm not sure whether this is a good idea. But leaving this zIndex at 100 made the poll bars appear over the Minds modal overlay. Maybe there's a better approach?